web-wallet/src/lib/containers/MigrateContract/tokenConfig.js

```diff
@@ -32,4 +32,5 @@ export const tokens = {
       name: "ERC-20",
     },
   },
+  unknown: null,
```

Comment:
This, and the type change below, is a bit weird.
Also we don't need to expand the type, but rather restrict the possible values received by the migrate contract (which you do later on).